packages/cursorless-vscode-e2e/src/suite/fixtures/scopes/javascript/name.assignment.scope

```diff
@@ -0,0 +1,24 @@
+const name = "Hello world";
+---
+
+[Content] = 0:6-0:10
+0| const name = "Hello world";
```

Comment:
The one thing I do find throwing me is the leading space. My brain keeps trying to see it as part of the document. Have we experimented with dropping it? Is it too busy?

Reply:
Yeah I guess they do look a bit cramped 😕. Would be cool to use those shading chars, but I guess prob not introducing strange unicode symbols for. Let's leave it for now; hopefully my brain will just adjust